content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,155 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+This pages describes how to install Kubeflow standalone components or Kubeflow Platform using package
+distributions or raw manifests. Check [the introduction guide](/docs/started/introduction) to
+understand what are Kubeflow standalone components and what is Kubeflow Platform.
 
-Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
-Operators can choose what is best for their users, there is no requirement to deploy every component.
+## Installing Kubeflow
 
-Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
-[Architecture](/docs/started/architecture/) pages.
+You can install Kubeflow components using one of these methods:
 
-## How to install Kubeflow?
+- [**Install Kubeflow Components Standalone**](#install-kubeflow-components-standalone)
+- [**Install Kubeflow Platform from Packaged Distributions**](#install-kubeflow-platform-from-packaged-distributions)
+- [**Install Kubeflow Platform from Raw Manifests**](#install-kubeflow-platform-from-raw-manifests) <sup>(advanced users)</sup>
 
-Anywhere you are running Kubernetes, you should be able to run Kubeflow.
-There are two primary ways to install Kubeflow:
+### Install Kubeflow Components Standalone
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+Kubeflow components can be deployed as standalone applications. You can integrate those components
+to your existing AI/ML platform. This is the easiest method to get started with Kubeflow ecosystem
+since those components usually don't require additional management tools used in Kubeflow Platform.
```

Comment:
I added this:
```
Some components in the [Kubeflow ecosystem](/docs/started/architecture/#conceptual-overview) may be
deployed as standalone services, without the need to install the full platform. You might integrate
these services as part of your existing AI/ML platform or use them independently.
```

Regarding this comment:

>We should also not suggest that this is "easier" because that is not necessarily the case, and will depend on the user and their use-case.


@james-jwu and @jbottum proposed to add this since typical Kubeflow users won't see any problems with installing Kubeflow components standalone, and unlikely will get blocked during installation phase. That will help them to explore Kubeflow ecosystem and see the value of individual components. Even with Kubeflow Pipelines @james-jwu told us that users don't see any problems with installing this standalone.

 cc @kubeflow/kubeflow-steering-committee @kubeflow/wg-pipeline-leads

Reply:
Renamed it to `an easier method`. 
@jbottum This link should work in preview mode.